docs/node-js/knex/README.md

```diff
@@ -242,10 +242,10 @@ const { NodeTracerProvider } = require("@opentelemetry/node");
 const { BatchSpanProcessor } = require("@opentelemetry/tracing");
 const {
   TraceExporter,
-} = require("@google-cloud/opentelemetry-cloud-trace-exporter");
+} = require("@opentelemetry/opentelemetry-cloud-trace-exporter");
```

Comment:
This can be `@google-cloud/opentelemetry-cloud-trace-exporter`. There is no `@opentelemetry/opentelemetry-cloud-trace-exporter`